go.mod

```diff
@@ -1,6 +1,6 @@
 module go.mongodb.org/atlas-sdk/v20240805004
 
-go 1.21
+go 1.22.0
```

Comment:
i think it's better to just keep go 1.22. in go.mod, specific 1.22.0 , etc. would be in toolchain but I don't think that's a best practice

Reply:
I haven't found clear guidance on full version here version language level and binary in the toolchain and in general what I've seen is no matter what you use the experience is really bad